scripts/sysman/fabric.yml

```diff
@@ -185,6 +185,26 @@ members:
       name: "txCounter"
       desc: "[out] Monotonic counter for the number of bytes transmitted (sum of all lanes). This includes all protocol overhead, not only the GPU traffic."
 --- #--------------------------------------------------------------------------
+type: struct
+desc: "Fabric Port Error Counters"
+class: $sFabricPort
+name: $s_fabric_port_error_counters_t
+base: $s_base_properties_t
```

Comment:
Please add the corresponding `FABRIC_PORT_ERROR_COUNTERS` enumeration to the `$s_structure_type_t` enum.

Reply:
Updated strucutre type and added that to the enum in common.yml